docs-gen/content/introduction/overview.md

```diff
@@ -40,7 +40,7 @@ The figure below shows an example snapshot of a generated tree of the
 specification. The leafs contain the actual information as shown in the figure.
 Before going into detail of the specification, let's dig deeper into taxonomies.
 
-![Example tree](/vehicle_signal_specification/images/tree.png?classes=shadow&width=60pc)
+![Example tree](https://covesa.github.io/vehicle_signal_specification/images/tree.png?classes=shadow&width=60pc)
```

Comment:
I think adding the "full path" shall not be necessary, i.e. this tree is visible today if you look at https://covesa.github.io/vehicle_signal_specification/introduction/overview/ (but there shall be no md as used in the changed like above)

Reply:
A side note - after working a bit with the documentation for VSS 4.0. Paths like "/vehicle_signal_specification/images/tree.png" reflects location after Hugo-generation, when for instance looking at https://covesa.github.io/vehicle_signal_specification/introduction/overview/. They do not work out of the box if looking at github source at https://github.com/COVESA/vehicle_signal_specification/blob/master/docs-gen/content/introduction/overview.md.

Changing to full link like here makes it work for both raw markdown and generated Hugo, but would give problems if we later would like to publish multiple versions of documentation (like 4.0, 3.1 and so on) in parallel, as it then will point to the same location for all versions. There might be some alternative ways to solve this, but it does not seem to be that easy.